### PR TITLE
Issue #1011 Fix Adding developer to sudoers when use v1.3.6

### LINKED ISSUE #1011: Adding developer to sudoers fails when using 3.6.x version of OpenShift
Looks like we have some code regression the way `minishift_kubeconfig` created and this make minishift failed during adding `developer` user to sudoers.

@jorgemoralespou 

```
$ ./minishift start --openshift-version v3.6.0-alpha.2
[...]
Image pull complete
OpenShift server started.

The server is accessible via web console at:
    https://192.168.42.71:8443

You are logged in as:
    User:     developer

To login as administrator:
    oc login -u system:admin

Error during post cluster up configuration: Unable to add sudoer role
```

### PR DESCRIPTION
**Pull request by praveenkumar**
Fix #1011

### CODE REVIEW

**Comment by budhrg** [comment]
I am fine with this change as long as the system admin IP will be 127.0.0.1 and as per this comment https://github.com/minishift/minishift/issues/1011#issuecomment-308483631 by Cesar, it looks like it will be.

LGTM :+1:

File: pkg/minikube/kubeconfig/config.go [hunk 1]
```diff
@@ -83,7 +84,7 @@ func CacheSystemAdminEntries(systemEntriesConfigPath, clusterName string) error
 
 	userName := fmt.Sprintf("system:admin/%s", clusterName)
 	for k, v := range config.Users {
-		if v.Name == userName {
+		if v.Name == userName || v.Name == "system:admin/127-0-0-1:8443" {
```
Comment (budhrg) [inline pkg/minikube/kubeconfig/config.go]: :+1:

**Comment by hferentschik** [comment]
@praveenkumar I this still WIP?

**Comment by budhrg** [comment]
Probably writing tests. :smile:  @hferentschik

**Comment by praveenkumar** [comment]
> I this still WIP?

I am having it WIP till get feedback from @csrwng

**Comment by hferentschik** [comment]
> I am having it WIP till get feedback from @csrwng

+1

**Comment by csrwng** [comment]
The change LGTM

**Comment by hferentschik** [comment]
@csrwng thanks for taking a look

**Comment by hferentschik** [comment]
Merged, thanks

**Pull request merged**